manager/src/main/java/com/redhat/service/bridge/manager/BridgesService.java

```diff
@@ -16,9 +17,31 @@
 
     Bridge getBridge(String id);
 
-    Bridge getBridge(String id, String customerId);
-
-    Bridge getReadyBridge(String bridgeId, String customerId);
+    /**
+     * Provides Bridge instance for given id/name. It throws `ItemNotFoundException` if bridge instance not found.
+     * 
+     * @param bridgeIdentifier It could be a bridge Id or bridge name.
+     * @param customerId Customer id.
+     * @return Bridge instance.
+     */
+    Bridge getBridgeByBridgeIdentifier(String bridgeIdentifier, String customerId);
+
+    /**
+     * Provide Bridge instance for given bridge id.
+     * 
+     * @param id Bridge id.
+     * @param customerId Customer id.
+     * @return Optional Bridge instance.
+     */
+    Optional<Bridge> getBridgeByIdAndCustomerId(String id, String customerId);
```

Comment:
Let's user either `Id` or `Identifier` as they mean the same thing. I'd go for `Id` as it's shorter and well-known.

Reply:
`Id` & `Identifier` are not same. `Id` corresponds to only `Bridge ID` but Identifier could be an `Bridge Id` or a `Bridge Name`.

Reply:
Same consideration as [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/422#discussion_r823698003)

Reply:
@vaibhavjainwiz "Identifier" means something that identifies unequivocally a row in the database. 

As the name is not in the primary key, name is not an identifier. I'd change it, it's confusing

Reply:
I changed it to `getBridgeByIdOrNameAndCustomerId`.